src/ClassicUO.Client/LegionScripting/docs/API.py

```
@@ -1377,6 +1382,36 @@
+     This includes trees, vegetation, buildings, and other non-movable scenery.
+     Example:
+     ```py
+     statics = API.GetStaticsAt(1000, 1000)
+     for static in statics:
+         API.Print(f"Static ID: {static.ID}, Z: {static.Z}")
+     ```
+    
+    """
+    pass
+
+def GetStaticsInArea(x1: int, y1: int, x2: int, y2: int) -> list[Any]:
```

Ruff (0.12.2)

Undefined name `Any`

(F821)